politeiawww/www.go

```diff
@@ -979,12 +981,45 @@ func _main() error {
 		log.Errorf("LoadInventory: %v", err)
 	}
 
-	// We don't persist connections to generate a new key every
-	// time we restart.
-	csrfKey, err := util.Random(32)
+	// Load or create new CSRF key
+	log.Infof("Load CSRF key")
+	csrfKeyFilename := filepath.Join(p.cfg.DataDir, "csrf.key")
+	fCSRF, err := os.Open(csrfKeyFilename)
+	if err != nil {
+		if os.IsNotExist(err) {
```

Comment:
if !os.IsNotExist(err) { return err }

then you get to lose indentation.

Reply:
Too confusing to read IMO. This is one of those exceptions that I'll take indentation over complexity.